phpBB/viewtopic.php

```diff
@@ -2283,6 +2313,14 @@
 $vars = array('page_title', 'topic_data', 'forum_id', 'start', 'post_list');
 extract($phpbb_dispatcher->trigger_event('core.viewtopic_modify_page_title', compact($vars)));
 
+$template->assign_vars(array(
+	"PHPBB_TOPIC_TITLE"=>$phpbb_topic_title,
```

Comment:
please add spaces around the `=>` and replaces the double quotes by simple quotes